src/lib/features/client-feature-toggles/client-feature-toggle.controller.ts

```diff
@@ -162,6 +165,32 @@ export default class FeatureController extends Controller {
     private async resolveFeaturesAndSegments(
         query?: IFeatureToggleQuery,
     ): Promise<[FeatureConfigurationClient[], IClientSegment[]]> {
+        if (this.flagResolver.isEnabled('deltaApi')) {
+            const features =
+                await this.clientFeatureToggleService.getClientFeatures(query);
+
+            const segments =
+                await this.clientFeatureToggleService.getActiveSegmentsForClient();
+
+            try {
+                const featuresSize = this.getCacheSizeInBytes(features);
+                const segmentsSize = this.getCacheSizeInBytes(segments);
+                this.clientFeaturesCacheMap.set(
```

Comment:
Saving the current key into cache.

Reply:
Again: can query be undefined here (because we force it with `!` later)?

So is this the size of the query *without* delta or with?

Reply:
Ah, I see now. This is specified in the store footprint method. This is for features.